app/assets/javascripts/request_service.js

```diff
@@ -0,0 +1,34 @@
+var RequestService = function() {
```

Comment:
One of the super powerful things in JavaScript is that you can just whip up objects on the fly without having to spin up a class like you would in Ruby.